src/pymovements/gaze/_utils/parsing.py

```diff
@@ -48,7 +48,6 @@
             r'\s+(?P<day>\d\d?)\s+(?P<time>\d\d:\d\d:\d\d)\s+(?P<year>\d{4})\s*'
         ),
         r'\*\*\s+(?P<version_2>EYELINK.*)',
-        r'MSG\s+\d+[.]?\d*\s+DISPLAY_COORDS\s*=?\s*(?P<resolution>.*)',
```

Comment:
instead of removing this regex completely, can you add the parsed info to the metadata dictionary? the regex group name and the metadata key can be simply `DISPLAY_COORDS`.

Reply:
Added it back, but as `(left, top, right, bottom)` instead of `(width, height)` (otherwise I think the name `DISPLAY_COORDS` seems confusing)